DotNet/LinkAdmin.BFF/Infrastructure/Extensions/Security/InitializeSecurity.cs

```diff
@@ -77,7 +79,36 @@ public static IServiceCollection AddLinkGatewaySecurity(this IServiceCollection
                 {
                     options.Cookie.Domain = configuration.GetValue<string>("Authentication:Schemas:Cookie:Domain");
                 }
-                    
+
+                options.Events.OnSigningIn = context =>
+                {
+                    //get claims principal
+                    if (context.Principal?.Identity is not ClaimsIdentity claimsIdentity)
+                    {
+                        return Task.CompletedTask;
+                    }
+                    // Define the claim types to keep
```

Comment:
Not necessary for this commit... But is there a reason to make this configurable on the service in the future? So that the appsettings.json file can drive what are `allowedClaims`?